src/relay/op/tensor/transform.cc

```diff
@@ -1553,6 +1553,52 @@ RELAY_REGISTER_OP("meshgrid")
     .set_attr<FTVMCompute>("FTVMCompute", MeshgridCompute)
     .set_attr<TOpPattern>("TOpPattern", kInjective);
 
+TVM_REGISTER_NODE_TYPE(SparseReshapeAttrs);
+
+bool SparseReshapeRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
+                      const TypeReporter& reporter) {
+  // types: [sparse_indices, sparse_values, result]
+  // ICHECK_EQ(types.size(), 3);
+  auto sparse_indices = types[0].as<TensorTypeNode>();
+  const auto* param = attrs.as<SparseReshapeAttrs>();
+  CHECK(param != nullptr);
+  Array<PrimExpr> new_sparse_indices_shape{sparse_indices->shape[0],
+                                           static_cast<int>((param->new_shape).size())};
+  reporter->Assign(types[2], TensorType(new_sparse_indices_shape, sparse_indices->dtype));
+  return true;
+}
+
+Array<te::Tensor> SparseReshapeCompute(const Attrs& attrs, const Array<te::Tensor>& inputs,
+                                       const Type& out_type) {
+  // ICHECK_EQ(inputs.size(), 2);
```

Comment:
uncomment or remove. If you keep it, it would be great to have an error message too.

Reply:
Done.